src/parser/shaman/elemental/integrationTests/example.test.ts.snap

```diff
@@ -5988,7 +5988,7 @@ exports[`Elemental Shaman integration test: example log matches the checklist sn
                   }
                 >
                    
-                  0.00%
+                  6.67%
                    
                    
```

Comment:
Elemental shamans being changed - is that intended?

Reply:
not sure. maybe they use Early Dot Refreshes. Ill double check to see if anything changed functionally in there

Reply:
So, the snapshot they are using is from Eternal Palace (Orgozoa), and when i look in the snapshot to get the WCL report so i can load up that fight and see what is going on, the report doesnt exist anymore, and has a weird link.

https://www.warcraftlogs.com/reports/TEST/#fight=18&source=23&type=damage-done

So i dont know what happened to make their snapshot change, but i also cant dig into that log to check ...

Reply:
Well that WCL link would've never worked 😄 
Let me upload a fresh one to them, and then we can compare changes between them properly.

Reply:
Seems like with the new integration test nothing changed due to my changes